cmd/eksctl-anywhere/cmd/options.go

```diff
@@ -23,6 +27,17 @@ func (c clusterOptions) mountDirs() []string {
 	return dirs
 }
 
+func mountPrivateKeyFile() (string, bool) {
+	privateKeyFilePath, err := git.GetTokenFromEnv(EksaGitPrivateKeyTokenEnv)
```

Comment:
It's a bit weird to get a path from a method called GetToken
Can we just read the env var normally? Or just rename that method to ReadEnvVar or something like that
Which unless there is some special git logic, should be in a different package

Reply:
yeah we should rename that method -- it was called 'GetTokenFromEnv' bc it was used for the Github Personal Access Token, but we should use some generic accessor